Drivers/SensorFusion/Src/sensor_fusion.cpp

```diff
@@ -488,23 +495,22 @@ SFError_t SF_GenerateNewResult()
 {
     SFError_t SFError;
     SFError.errorCode = 0;
-    xSemaphoreTake(SF_mutex, portMAX_DELAY);
+    SFAttitudeOutput_t attitudeOutput;
+    memset(&attitudeOutput, 0, sizeof(attitudeOutput));
+    SFPathOutput_t pathOutput;
+    memset(&pathOutput, 0, sizeof(pathOutput));
```

Comment:
Why is this `memset` needed?

Reply:
to not assign uninitiated value into other variables. to eliminate the warning that prompt from compile